test/csuite/random_abort/main.c

```diff
@@ -832,8 +836,11 @@ main(int argc, char *argv[])
      */
 
     /* Clean up the test directory. */
-    if (ret == EXIT_SUCCESS && !preserve)
-        testutil_clean_test_artifacts(home);
+    if (ret == EXIT_SUCCESS && !preserve) {
+        /* Current working directory is home (aka WT_TEST) */
+        testutil_assert_errno(getcwd(buf, sizeof(buf)) != NULL);
+        testutil_clean_test_artifacts(buf);
```

Comment:
In `random_directio`, you also call `testutil_remove(buf);`, do we need it here as well?

Reply:
it is eventually called at line 855 when it checks if the preserve flag is set or not.